src/client/components/leaderboard/leaderboard.js

```diff
@@ -8,8 +8,18 @@ class Leaderboard extends React.Component {
     names: PropTypes.any.isRequired,
     cards: PropTypes.any.isRequired,
     playerID: PropTypes.any.isRequired,
+    passed: PropTypes.any.isRequired,
   };
   render() {
+    let passed = this.props.passed;
+    function hasPassed(_idx) {
```

Comment:
In a more descriptive way this could be written as
```js
function hasPassed(userId) {
  return this.props.passedUsers.includes(userId.toString());
}
```